src/components/summary-list/summary-list.yaml

```diff
@@ -27,15 +27,15 @@ params:
     type: string
     required: false
     description: Classes to add to the value wrapper
+  - name: actions.classes
```

Comment:
Oops, you're right we had this documented incorrect all this time, thanks a bunch!